### PR TITLE
New section `Documentation generator` + entry

### PR DESCRIPTION
**Pull request by pguth**
+ TOC entry,  
+ the new section itself,
+ new entry `flatdoc`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ Table of Contents
   - [IDE/Tools](#idetools)
   - [Continuous Integration](#continuous-integration)
   - [Bug and Task Trackers](#bug-and-task-trackers)
+  - [Documentation generator](#documentation-generator)
```
Comment (nodiscc) [inline README.md]: Please use title casing and plural (`Documentation Generators`)

File: README.md [hunk 2]
```diff
@@ -504,6 +505,11 @@ CMS are a practical way to setup a website with many features. CMS often come wi
   * [Tinyissue](https://github.com/mikelbring/tinyissue) - Simple Issue Tracking for Teams `MIT`
   * [Tracks](http://getontracks.org/) -  a web-based application to help you implement David Allen’s [Getting Things Done™](http://wikipedia.org/wiki/Getting_Things_Done) methodology. - `GPLV2`
 
+### Documentation generator
+
+  * [Flatdoc](http://ricostacruz.com/flatdoc/) - a small JavaScript file that fetches Markdown files and renders them as full pages `MIT`
```
Comment (nodiscc) [inline README.md]: I know it's redundant, but can you add `Javascript` as language?

**Comment by pguth** [comment]
Done (^_^) @nodiscc

File: README.md [hunk 3]
```diff
@@ -52,6 +52,7 @@ Table of Contents
   - [IDE/Tools](#idetools)
   - [Continuous Integration](#continuous-integration)
   - [Bug and Task Trackers](#bug-and-task-trackers)
+  - [Documentation Generators](#documentation-generator)
```
Comment (nodiscc) [inline README.md]: You forgot to update the anchor `(#documentation-generators)`

**Comment by nodiscc** [comment]
Thanks @pguth while we're at it, could you move daux.io , Docstore, markdown-tree and mdwiki to that section? They're currently in the wiki section

@Kickball what do you think about the new doc generators section?

**Comment by pguth** [comment]
No problemo. Collected and moved @nodiscc

**Comment by nodiscc** [comment]
Thanks a lot

**Pull request merged**